src/ziggurat/streams.clj

```diff
@@ -154,6 +154,25 @@
   (.transformValues stream-builder (header-transformer-supplier) (into-array [(.name (store-supplier-builder))])))
 
 (declare stream)
+(def stream-map {})
```

Comment:
We should not hold the map in a variable. 

What I meant was this: `defstate` streams currently holds a vector of all streams objects. Change that to hold a map of `topic-entity` to a `KafkaStreams` object.